src/main/java/io/jenkins/plugins/appcenter/task/internal/UploadAppToResourceTask.java

```diff
@@ -1,10 +1,13 @@
 package io.jenkins.plugins.appcenter.task.internal;
 
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobClientBuilder;
 import hudson.FilePath;
 import hudson.model.TaskListener;
 import io.jenkins.plugins.appcenter.AppCenterException;
 import io.jenkins.plugins.appcenter.AppCenterLogger;
 import io.jenkins.plugins.appcenter.api.AppCenterServiceFactory;
+import io.jenkins.plugins.appcenter.api.UploadService;
```

Comment:
My IDE tells me this import is unused. Is this a problem with my setup or can we remove this import?